pkg/cmd/drt/scripts/tpcc_init.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The Cockroach Authors.
+#
+# Use of this software is governed by the CockroachDB Software License
+# included in the /LICENSE file.
+
+
+set -o pipefail
+
+TPCC_DB=cct_tpcc
+export ROACHPROD_GCE_DEFAULT_PROJECT=cockroach-drt
+export ROACHPROD_DNS="drt.crdb.io"
+./roachprod sync
+sleep 20
+PGURLS=$(./roachprod pgurl drt-scale:1-150 | sed s/\'//g)
```

Comment:
nit: passing only cluster name `drt-scale` would give pgurl for all nodes. In case if we decide to not have exact 150 nodes.